tools/src/main/scala/org/apache/spark/tools/GenerateMIMAIgnore.scala

```diff
@@ -39,12 +39,14 @@ import org.clapper.classutil.ClassFinder
 object GenerateMIMAIgnore {
   private val classLoader = Thread.currentThread().getContextClassLoader
   private val mirror = runtimeMirror(classLoader)
+  // SPARK-13920: MIMA checks should apply to @Experimental and @DeveloperAPI APIs
+  private val ignoreExpDevApi = false
```

Comment:
Rather than adding an on/off switch, I'd just go ahead and completely remove these methods.